cmd/minikube/cmd/config/validations.go

```diff
@@ -59,6 +58,35 @@ func IsValidURL(name string, location string) error {
 	return nil
 }
 
+func IsURLExists(name string, location string) error {
+	parsed, err := url.Parse(location)
+	if err != nil {
+		return fmt.Errorf("%s is not a valid URL", location)
+	}
+
+	// we can only validate if local files exist, not other urls
+	if parsed.Scheme != "file" {
+		return nil
+	}
+
+	// chop off "file://" from the location, giving us the real system path
+	sysPath := string([]rune(location[len("file://"):]))
```

Comment:
This works, but the run casting is a bit weird. Is it possible to use `parsed.Path`?

If not, these alternatives should work:
  * `strings.TrimPrefix("file://")`
  * `location[len("file://"):])`